web/src/search/input/MonacoQueryInput.tsx

```diff
@@ -69,7 +69,36 @@ function addSouregraphSearchCodeIntelligence(
     return subscriptions
 }
 
-const NOOP_KEYBINDINGS = [Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.KEY_F, Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.Enter]
+/**
+ * HACK: this interface and the below type guard are used to free default Monaco
+ * keybindings (such as cmd + F, cmd + L) by unregistering them from the private
+ * _standaloneKeybindingService.
```

Comment:
So the underscore doesn't make text italic

```suggestion
 * `_standaloneKeybindingService`.
```